src/QsCompiler/BondSchemas/BondSchemaTranslator.cs

```diff
@@ -536,9 +536,9 @@ private static QsTypeItem ToBondSchema(this SyntaxTree.QsTypeItem qsTypeItem)
             {
                 Origin = qsTypeParameter.Origin.ToBondSchema(),
                 TypeName = qsTypeParameter.TypeName,
-                Range = qsTypeParameter.Range.IsNull ?
-                    null :
-                    qsTypeParameter.Range.Item.ToBondSchema()
+#pragma warning disable 618 // QsTypeParameter.Range is obsolete.
```

Comment:
Actually, why  do we need to keep this? Isn't this what gets serialized; i.e. can't we drop this and just make sure that the corresponding item in already compiled old dlls is ignored when deserializing it?

Reply:
Well, we could drop the ranges during serialization, but that is technically a breaking change if anyone is relying on them in a deserialized syntax tree (e.g. a rewrite step that looks at references in the AST). Probably no one would be affected by it, but I wanted to continue fully supporting the old ranges until they are removed.

Reply:
Or do you mean that since the old behavior was to strip range information during deserialization, then this wouldn't be a breaking change?

Reply:
I think this might not actually break anything if we remove it, and people are supposed to be using the compiler methods to load the syntax tree rather than rely on the serialization direction. So as long as that keeps working for old dlls, we are good to remove this.